routes/v1.js

```diff
@@ -127,7 +131,20 @@ function addRoutes(app, peliasConfig) {
     isPipServiceEnabled, not(hasRequestErrors), not(hasResponseData)
   );
 
+  // defines whether to skip libpostal and control should be switched to intersection processing
+  const intersectionParserShouldExecute = all (
+    isIntersectionLayer,
+    not(hasRequestErrors)
+  );
+
+  const intersectionQueryShouldExecute = all (
+    isIntersectionLayer,
+    not(hasRequestErrors)
+  );
+
   const libpostalShouldExecute = all(
+    //not(isIntersectionLayer),
```

Comment:
If this and the following few commented out lines aren't needed please remove them.